ethclient/ethclient.go

```diff
@@ -24,7 +24,7 @@ import (
 	"fmt"
 	"math/big"
 
-	"github.com/maticnetwork/bor"
+	ethereum "github.com/maticnetwork/bor"
```

Comment:
maybe rename everything to bor? To have consistency

Reply:
Do we require this change
just making changes in this file will not be enough
there are around 24 files which imports `ethereum "github.com/maticnetwork/bor"`
we need to make changes there too
@vaibhavchellani

Reply:
Planning to do this in another separate PR.